hydra-node/json-schemas/api.yaml

```diff
@@ -1395,7 +1395,7 @@ components:
       required:
```

Comment:
We remove sequence numbers on messages coming from the api server that do not come from `StateEvent`. For example: `Greetings`, `InvalidInput`, `CommandFailed`, `PostTxOnChainFailed` etc.